webpack-plugin/src/index.ts

```diff
@@ -313,7 +313,7 @@ function createLoaderRules(
 	};
 	return [
 		{
-			test: /esm[/\\]vs[/\\]editor[/\\]editor.(api|main).js/,
+			test: /esm[/\\]vs[/\\]editor[/\\]editor.main.js/,
```

Comment:
I wonder, is this a breaking change for all existing users which were importing `'monaco-editor/esm/vs/editor/editor.api'`?

Reply:
Yes, I commented it in commit message:

BREAKING CHANGES:
if a project contains only these imports for creating editor:
```
import * as monaco from 'monaco-editor/esm/vs/editor/editor.api.js';
const editor = monaco.editor.create(...);
```
`monaco-editor-webpack-plugin` will not add features and languages. When creating editor,
 monaco must be imported from package itself:
 ```
 import * as monaco from 'monaco-editor';
 const editor = monaco.editor.create(...);